files/ansible/templates/lmod-ohpc1.3.1

```diff
@@ -0,0 +1,69 @@
+-- -*- lua -*-
+-- created: Tue Oct 30 17:46:11 2018 --
+-- Lmod 7.7.14  2017-11-16 16:23 -07:00
+_ModuleTable_ = {
```

Comment:
This looks like a horrible hack... Why do we need this?

This test with openHPC is looking like a terrible mistake.